src/coreclr/vm/gcenv.ee.cpp

```diff
@@ -91,6 +91,10 @@ VOID GCToEEInterface::AfterGcScanRoots (int condemned, int max_gen,
     Interop::OnAfterGCScanRoots(sc->concurrent);
 }
 
+#if !defined(TARGET_X86) || defined(TARGET_UNIX)
+#define USE_STACK_LIMIT
```

Comment:
Can we just use `USE_FEF` here instead of this?

Reply:
Sure